### ISSUE TITLE
GAN Loss and Discriminator Loss are abnormal.

### ISSUE THREAD

**Issue by DragonisCV** [CompVis/latent-diffusion#383]
I tried to reproduce  [VAE](https://github.com/CompVis/latent-diffusion/blob/main/configs/autoencoder/autoencoder_kl_64x64x3.yaml) on ImageNet, but during the training process, I found that the discriminator loss kept increasing. When the discriminator is added to the training, the overall loss continues to decrease and becomes a very small negative number (around -1e6).
What could be the possible reasons? All hyper-parameters are set according to the config provided.
The following is the training curve showing the changes in all losses, including disc_loss, gan_loss, logit_fake, and logit_real. Is this phenomenon normal？

![image](https://github.com/user-attachments/assets/06fd5f99-1590-4ce6-8603-68006984f6d4)
